includes/autoloaders.inc.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+
+/**
+ * @file includes/autoloaders.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @ingroup index
+ *
+ * @brief Registers custom autoloaders 
+ */
+
+// registering autoloader function for OJS namespace
+spl_autoload_register(function($class) {
+	$prefix = 'OJS\\';
```

Comment:
Not sure if you saw my second comment, but I do think it would be better to keep the two namespaces separate, just that the OJS-related part needs to be kept in that app's repository.

We should never have a class name that appears in both places, so I don't really care about the order.